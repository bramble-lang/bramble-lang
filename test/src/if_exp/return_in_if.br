fn my_main() -> i64 {
    root::std::io::writei64ln(test(0));
    root::std::io::writei64ln(test(20));
    return 0;
}

fn test(x: i64) -> i64 {
    if (x < 10) {
        return 0;
    } else {
        return 1;
    };

    return -1;
}