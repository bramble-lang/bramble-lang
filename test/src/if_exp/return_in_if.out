Error: Parser: L9: Expected }, but found return
